src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -87,10 +106,110 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityProps> = ({
   }, [foundPoolAsset])
 
   const rune = useAppSelector(state => selectAssetById(state, thorchainAssetId))
+  const runeAsset = useAppSelector(state => selectAssetById(state, thorchainAssetId))
 
   const [poolAsset, setPoolAsset] = useState<Asset | undefined>(foundPoolAsset)
   const [userlpData, setUserlpData] = useState<UserLpDataPosition | undefined>()
+  const [runeAccountId, setRuneAccountId] = useState<AccountId | undefined>()
   const [percentageSelection, setPercentageSelection] = useState<number>(50)
+  const [shareOfPoolDecimalPercent, setShareOfPoolDecimalPercent] = useState<string | undefined>()
+
+  // Virtual as in, these are the amounts if removing symetrically. But a user may remove asymetrically, so these are not the *actual* amounts
+  // Keeping these as virtual amounts is useful from a UI perspective, as it allows rebalancing to automagically work when switching from sym. type,
+  // while using the *actual* amounts whenever we do things like checking for asset balance
+  const [virtualAssetCryptoLiquidityAmount, setVirtualAssetCryptoLiquidityAmount] = useState<
+    string | undefined
+  >()
+  const [virtualAssetFiatLiquidityAmount, setVirtualAssetFiatLiquidityAmount] = useState<
+    string | undefined
+  >()
+  const [virtualRuneCryptoLiquidityAmount, setVirtualRuneCryptoLiquidityAmount] = useState<
+    string | undefined
+  >()
+  const [virtualRuneFiatLiquidityAmount, setVirtualRuneFiatLiquidityAmount] = useState<
+    string | undefined
+  >()
+  const [cryptoAssetBalance, setCrytoAssetBalance] = useState<string | undefined>()
```

Comment:
And `setCrytoRuneBalance` too
```suggestion
  const [cryptoAssetBalance, setCryptoAssetBalance] = useState<string | undefined>()
```

Reply:
Also we probably want to use a different terminology here? i.e balance would refer to the wallet balance, perhaps `stakedAmountCryptoBaseUnit` ?

Or even better, remove these and don't set anything in state since this is simply destructuring and renaming from userLpData and setting properties such as `userLpData.underlyingAssetAmountCryptoPrecision` as state values?

Reply:
gm https://github.com/shapeshift/web/pull/6175/commits/c13578dbbaf318dd43792431b655e7f7601d26a5

Reply:
Do we want to remove these altogether here?

Reply:
That's reasonable 🙏  https://github.com/shapeshift/web/pull/6175/commits/ce65270269aca06920532f2bb741e003db7495dd